web-client/integration-tests/adcClerkViewsMessagesJourney.test.js

```diff
@@ -0,0 +1,296 @@
+import { MESSAGE_QUEUE_TYPES } from '../../shared/src/business/entities/EntityConstants';
+import { applicationContext } from '../src/applicationContext';
+import { createNewMessageOnCase } from './journey/createNewMessageOnCase';
+import { formattedMessages } from '../src/presenter/computeds/formattedMessages';
+import {
+  getUserMessageCount,
+  loginAs,
+  refreshElasticsearchIndex,
+  setupTest,
+  uploadPetition,
+} from './helpers';
+import { runCompute } from 'cerebral/test';
+import { showSortableHeaders as showSortableHeadersComputed } from '../src/presenter/computeds/showSortableHeaders';
+import { withAppContextDecorator } from '../src/withAppContext';
+
+const formattedMessagesComputed = withAppContextDecorator(
+  formattedMessages,
+  applicationContext,
+);
+
+const cerebralTest = setupTest();
+
+describe('ADC Clerk Views Messages Journey', () => {
+  beforeAll(() => {
+    jest.setTimeout(30000);
+  });
+
+  afterAll(() => {
+    cerebralTest.closeSocket();
+  });
+
+  MESSAGE_QUEUE_TYPES.forEach(messageQueue => {
+    const testAdcId = '6805d1ab-18d0-43ec-bafb-654e83405416';
+    const petitionsClerkId = '4805d1ab-18d0-43ec-bafb-654e83405416';
+    let beforeInboxMessageCount = 0;
+    let beforeOutboxMessageCount = 0;
+    let beforeCompletedMessageCount = 0;
+
+    loginAs(cerebralTest, 'adc@example.com');
+    it('get before counts for all section message boxes', async () => {
+      await getUserMessageCount(cerebralTest, 'inbox', messageQueue);
```

Comment:
Could we make use of the value returned from getUserMessageCount like on line 109 to get the length of the  messages in the provided messageQueue?

Reply:
Yes. We've made that change.